### PR TITLE
FIX MKDIR

### PR DESCRIPTION
**Pull request by jhcpeixoto**

### CODE REVIEW

**Comment by jhcpeixoto** [comment]
image

File: .env.example [hunk 1]
```diff
@@ -1,3 +0,0 @@
-STORAGE_LOCATION=files/
-STORAGE_PROVIDER=file
```
Comment (Flam3rboy) [inline .env.example:2]: Why did you delete the example env file?

File: src/util/FileStorage.ts [hunk 2]
```diff
@@ -4,27 +4,27 @@ import { join } from "path";
...
-		path = join(process.env.STORAGE_LOCATION || "", path);
-		try {
-			const file = await fs.readFile(path);
-			// @ts-ignore
-			return file;
-		} catch (error) {
-			return null;
-		}
-	}
+  async get(path: string): Promise<Buffer | null> {
```
Comment (Flam3rboy) [inline src/util/FileStorage.ts:7]: Please do not reformat the file and use 4 width tabs

**Review by Flam3rboy — CHANGES_REQUESTED** [review]

File: src/util/FileStorage.ts (hunk 2 above)
Comment (jhcpeixoto) [reply src/util/FileStorage.ts]: FIXED

File: .env.example (hunk 1 above)
Comment (jhcpeixoto) [reply .env.example]: Done
